src/frontend/src/components/SearchBar/SearchBar.jsx

```diff
@@ -60,16 +61,16 @@ const useStyles = makeStyles((theme) => ({
       backgroundColor: theme.palette.secondary.dark,
     },
     '& * > .MuiSvgIcon-root': {
-      fontSize: '2rem',
+      fontSize: '3rem',
       color: theme.palette.text.primary,
     },
     margin: 0,
+    marginLeft: 15,
```

Comment:
15 what?  Needs a unit.

Reply:
I need to remove all changes to the SearchBar for that kind of stuff actually.